src/js/components/sharedComponents/Pagination.jsx

```diff
@@ -142,20 +156,26 @@ export default class Pagination extends React.Component {
 
         return (
             <div className="pagination">
-                <div className="results-text">
+                <div className="pagination__totals">
                     {resultsText}
                 </div>
                 <ul className="pager">
-                    <li className={pager.currentPage === 1 ? 'disabled' : ''}>
-                        <button onClick={() => this.setPage(pager.currentPage - 1)}>{`<`}</button>
+                    <li className={`pager__item ${pager.currentPage === 1 ? 'pager__item_disabled' : ''}`}>
```

Comment:
When buttons are disabled, can you set the `disabled` attribute on the button itself for accessibility? Also, the buttons still appear to be clickable at the moment, so you can go to negative pages.

Reply:
Set a disabled attribute on the buttons